src/manifold/include/manifold/manifold.h

```diff
@@ -37,22 +37,12 @@ class CsgLeafNode;
  *  @{
  */
 
-template <typename Precision>
+template <typename Precision, typename I = uint32_t>
```

Comment:
Honestly, does it hurt anything for runIndex and runOriginalID to be 64 bit? They're very short vectors anyway. Seems simpler and more fitting with the name `MeshGL64` for everything to be 64 bit instead of just some.

Reply:
Sure